### DISCUSSION TITLE
Simplified/cheaper trial deployment example config

### DISCUSSION THREAD

**Discussion by tograla** [aws-samples/aws-secure-environment-accelerator#729] [Q&A] [answered]
Hey Everyone!
How much would it cost, approximately, if I spun up an organization using the Accelerator and let it run for say a month? I presume that would be needed to get a familiar.

I'm considering trialing it on my personal account for learning purposes, I'm a solutions architect (associate), so I'm aware different services have different pricing model etc, some of them may be price'y if used wildly, many others come with a free-tier and I just rather know ahead of time to avoid painful surprises. 
I presume I'd experiment with my setup a little by deploying some skinny ec2's, lambda's etc. but that would obviously count toward my own spent.
Cheers!

**Comment by Brian969** [comment, marked as answer]
The current sample deployment(s) are a little expensive when used solely for testing in a personal account, yet typically only represent a very small percentage of a production customers AWS spend.  The current sample config files build a highly available and scalable multi-datacenter environment with hyperscale routing and enterprise grade security worldwide, something that would cost tens of millions of dollars on-premise and still not achieve the same results :-). 

We've had multiple requests for something more appropriate for a personal test account, yet still provides a good representation of the SEA functionality.  If you wait a week, I will try to publish something better suited to this requirement.

**Reply by ghost** [reply]
This would be really interesting to see! Been following the build for a while, we're interesting in building it out in our org.

**Reply by tograla** [reply]
I second that, and I can definetely wait. We're using Landing Zone at my company but this is quite complex service and quite a niche thing to learn at scale, so i thought of spinning up my own little set-up using Accelerator just to learn and boost confidence by actually observing a bigger environment in its entirety.
I do appreciate the warning about possible costs :)

**Reply by obriensystems** [reply]
I have a requirement on understanding/reverse-engineering sced.  Thank you for this accelerator, spinning it up - I’ll advise on the hourly finops shortly

**Reply by Brian969** [reply]
[Example 5](https://github.com/aws-samples/aws-secure-environment-accelerator/blob/main/docs/installation/customization-index.md#11-sample-accelerator-configuration-files) reduces costs considerably.  ~~Still working on an example removing the Fortigates (sorry, I've had little time).  Pending~~ v1.5.0 ~~(likely 4-weeks out) will~~ further simplified with new perimeter options including AWS Network Firewall in example 5 ~~, and Gateway Load Balancer (example 2, viariant 4)~~, and automated updating of the FE ALB with the latest backend ALB addresses, etc.
